packages/components/src/components/checkbox-wrapper/checkbox-wrapper.tsx

```diff
@@ -80,7 +85,18 @@ export class CheckboxWrapper {
 
   public render(): JSX.Element {
     validateProps(this, propTypes);
-    attachComponentCss(this.host, getComponentCss, this.hideLabel, this.state, this.input.disabled, this.theme);
+    attachComponentCss(
+      this.host,
+      getComponentCss,
+      this.hideLabel,
+      this.state,
+      this.input.disabled,
+      this.loading,
+      this.input.checked,
```

Comment:
Would be good to list the `input` parameters right after each other or eve pass the entire `input`.

Reply:
I changed the implementation and we don't pass `this.input.checked` anymore